pkg/operator/staticpod/controller/installer/installer_controller.go

```diff
@@ -244,6 +255,7 @@ func (c *InstallerController) createInstallerPod(currNodeState *operatorv1alpha1
 		fmt.Sprintf("--pod=%s", c.configMaps[0]),
 		fmt.Sprintf("--resource-dir=%s", "/etc/kubernetes/static-pod-resources"),
 		fmt.Sprintf("--pod-manifest-dir=%s", "/etc/kubernetes/manifests"),
+		fmt.Sprintf("--revision-history-limit=%d", operatorSpec.RevisionHistoryLimit),
```

Comment:
I wouldn't expect an installer pod to know this.  I would expect a separate binary to deal with pruning

Reply:
Why can't the installer handle cleaning up old installations? It created them so I don't think it's unreasonable for it to prune them too